tests/test_analyzer.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+from pymc_workflow_analyzer.analyzer import static_analyze
+from pymc_workflow_analyzer.report import static_report_generator, save_report
+
+def test_analyze_script():
+    script_path = 'sample_model.py'
+    expected_result = {
+        'imports': ['arviz', 'matplotlib.pyplot', 'numpy', 'pandas', 'pymc', 'xarray'], 
+        'distributions': ['HalfCauchy', 'Normal'],
```

Comment:
This is great!

Reply:
Thank you. Any suggestions for improvement you might have?

Reply:
now we are getting all the distributions